src/ThreeEditor/main.js

```diff
@@ -27,16 +27,16 @@ export function initEditor(container) {
 	container.appendChild(viewManager.dom);
 
 	const toolbar = new Toolbar(editor);
-	container.appendChild(toolbar.dom);
+	// container.appendChild(toolbar.dom);
 
 	const sidebar = new Sidebar(editor);
 	container.appendChild(sidebar.dom);
 
 	const menubar = new Menubar(editor);
-	container.appendChild(menubar.dom);
+	// container.appendChild(menubar.dom);
```

Comment:
why is this commented out ?

Reply:
This UI was being replaced and now i can delete it.